shell/pages/home.vue

```diff
@@ -287,9 +287,9 @@ export default {
         </div>
       </div>
 
-      <div class="row">
-        <div :class="{'span-9': showSidePanel, 'span-12': !showSidePanel }" class="col">
```

Comment:
If we're not using `showSidePanel` anymore the computed property can be removed. This also applies to `showCommercialSupport` and `showCommunityLinks`